lib/ansible/modules/cloud/hcloud/hcloud_datacenter_info.py

```diff
@@ -40,32 +42,35 @@
 """
 
 EXAMPLES = """
-- name: Gather hcloud datacenter facts
-  local_action:
-    module: hcloud_datacenter_facts
-- name: Print the gathered facts
+- name: Gather hcloud datacenter info
+  hcloud_datacenter_info:
+  register: output
+- name: Print the gathered info
   debug:
-    var: ansible_facts.hcloud_datacenter_facts
+    var: output
 """
 
 RETURN = """
-hcloud_datacenter_facts:
-    description: The datacenter facts as list
+hcloud_datacenter_info:
+    description:
+      - The datacenter info as list
+      - This module was called C(hcloud_datacenter_facts) before Ansible 2.9, returning C(ansible_facts) and C(hcloud_datacenter_facts).
```

Comment:
```suggestion
      - This module was called C(hcloud_datacenter_facts) before Ansible 2.9, returning C(hcloud_datacenter_facts) as C(ansible_facts).
```
You have to decide whether you want to change this in all places (or in none :) ).